it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/templates/af-blank-v2/.content.xml

```diff
@@ -4,7 +4,7 @@
     <jcr:content
         cq:lastModified="{Date}2021-08-31T15:15:47.862+05:30"
         cq:lastModifiedBy="admin"
-        cq:templateType="/conf/core-components-it/settings/wcm/template-types/af-blank-v2"
+        cq:templateType="/conf/core-components-it/settings/wcm/template-types/af-page-v2"
         jcr:description="AF v2 Template for Core component examples"
         jcr:primaryType="cq:PageContent"
         jcr:title="AF Blank V2"
```

Comment:
can we rename it as `Blank (Core Components)`

Reply:
Fixed.

Reply:
@nit23uec, Removed (Core Components), as suggested by Anurag, it was also coming from template type string as well.